scripts/ssl-manager.js

```diff
@@ -873,7 +936,9 @@ function SSLManager(config) {
                 "withExtIp='%(withExtIp)'",
                 "webroot='%(webroot)'",
                 "webrootPath='%(webrootPath)'",
-                "skipped_domains='%(skipped)'"
+                "skipped_domains='%(skipped)'",
```

Comment:
3 different naming styles in settings file

Reply:
discussed internally